hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -383,6 +383,14 @@
       run_opts: ["+use_spi_load_bootstrap=1", "+calibrate_usb_clk=1"]
       run_timeout_mins: 180
     }
+    {
+      name: chip_sw_inject_scramble_seed
+      uvm_test_seq: chip_sw_inject_scramble_seed_vseq
+      sw_images: ["//sw/device/tests/sim_dv:inject_scramble_seed:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+lc_at_prod=1", "+flash_program_latency=5", "+sw_test_timeout_ns=150_000_000"]
```

Comment:
I will create an image for prod lc_state soon. Sorry for the inconvenience.

Reply:
no worries!